usecase-1/src/Pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import Header from "../../Components/Header/Header";
+import "./HomePage.css";
+import Cards from "../../Components/Cards/Cards";
+import data from "../../data.js";
+// Importing Header and Cards Component
+export default function HomePage() {
+  return (
+    <div id="container">
+      <Header />
+      <div id="cards-container">
```

Comment:
Can we create another component (eg : CardsContainer) and add logic there to promote reusability ?